### DISCUSSION TITLE
不使用【part-key】，使用【part-name】的方式，最多可以创建多少个分区

### DISCUSSION THREAD

**Discussion by opendatamarket** [milvus-io/milvus#25851] [Q&A and General discussion] [answered]
Milvus version 2.2.11
Java SDK version  2.2.8 

If there are more than 5000 partitions, will it affect query efficiency, and how many are suitable to create at most?

**Comment by xiaofan-luan** [comment, marked as answer]
最多支持创建4096 partitions，
过多partition可能导致额外的内存占用
